build.gradle

```diff
@@ -40,24 +40,24 @@ java {
 }
 
 wrapper {
-    gradleVersion = '8.10'
+    gradleVersion = '8.12'
 }
 
 dependencies {
     dependencies {
-        implementation "org.springdoc:springdoc-openapi-data-rest:${sprindocAnnotationsVersion}"
-        implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
-        implementation "javax.validation:validation-api:${validationApiVersion}"
-        implementation "org.apache.commons:commons-lang3:${commonsLangVersion}"
+        api "org.springdoc:springdoc-openapi-starter-common:${springDocVersion}"
```

Comment:
I didn't manage to find any single reference on Spring in the code. If don use this library only to include transient Swagger annotations, please remove it and include `swagger-annotations` directly.

Reply:
fixed